test/lib/commands/config.js

```diff
@@ -3,16 +3,22 @@ const fs = require('fs/promises')
 const ini = require('ini')
 const tspawk = require('../../fixtures/tspawk')
 const t = require('tap')
+const { load: loadMockNpm } = require('../../fixtures/mock-npm')
+const { cleanCwd } = require('../../fixtures/clean-snapshot.js')
+const { version: npmVersion } = require('../../../package.json')
 
 const spawk = tspawk(t)
 
-const Sandbox = require('../../fixtures/sandbox.js')
```

Comment:
Did this file get deleted?

Reply:
Not yet! Just pushed a commit deleting it now.